### PR TITLE
[corrade/magnum/-plugins] Fix build, minor cleanup and minor warning fix in vcpkg CMake scripts

### PR DESCRIPTION
**Pull request by Squareys**
Hi all,

I removed two patches from the magnum/magnum-plugins ports each, made the feature to cmake flag conversion process consistent with the magnum/magnum-plugins packages.
Finally, `vcpkg_copy_pdbs.cmake` always printed an warning about a list of no dlls not having a matching pdb, this was a small error in the if condition which was using a no longer existent variable.

I updated the versions, let me know if I missed anything else.

Best,
Jonathan

### CODE REVIEW

**Comment by Squareys** [comment]
> After committing all other changes, the version database must be updated

I suppose this is because I updated the database with the commits rather than adding a separate commit at the end?

File: scripts/cmake/vcpkg_copy_pdbs.cmake [hunk 1]
```diff
@@ -65,7 +65,7 @@ function(vcpkg_copy_pdbs)
 
         set(ENV{VSLANG} "${vslang_backup}")
 
-        if(NOT unmatched_dlls_length STREQUAL "")
+        if(NOT dlls_without_matching_pdbs STREQUAL "")
```
Comment (PhoebeHui) [inline scripts/cmake/vcpkg_copy_pdbs.cmake:68]: Same fix with PR https://github.com/microsoft/vcpkg/pull/19369.

**Comment by PhoebeHui** [comment]
@Squareys, thanks for your PR!

**Review by PhoebeHui — APPROVED** [review]

File: scripts/cmake/vcpkg_copy_pdbs.cmake (hunk 1 above)
Comment (Squareys) [reply scripts/cmake/vcpkg_copy_pdbs.cmake]: Awesome 👍

**Comment by BillyONeal** [comment]
Thank you ❤

**Pull request merged**